mysql/mysqldump.go

```diff
@@ -221,6 +226,9 @@ func processConstraint(conv *internal.Conv, table string, constraint *ast.Constr
 		// NOT NULL and UNIQUE i.e. primary keys must be NOT NULL and UNIQUE.
 		// We preserve MySQL semantics and enforce NOT NULL and UNIQUE.
 		updateCols(conv, ast.ConstraintPrimaryKey, constraint.Keys, st.ColDefs, table)
+	case ast.ConstraintForeignKey:
+		st.ForeignKeys = append(st.ForeignKeys, toForeignKeys(conv, constraint))
+		updateCols(conv, ast.ConstraintForeignKey, constraint.Keys, st.ColDefs, table)
```

Comment:
The call to updateCols isn't needed since we've done all of the processing needed for foreign keys in the previous line -- and the call is in fact a no-op after the change to remove the switch case for foreign keys.
Suggest deleting it to keep code simple.

Reply:
yes, removed updateCols call as it is not handling foreign key case anymore, removed the same for postgres as well

Reply:
Thanks!